Lib/test/test_xml_etree.py

```diff
@@ -2680,18 +2680,51 @@ class Y(X, ET.Element):
         e = ET.Element('foo')
         e.extend(L)
 
-    def test_remove_with_mutating(self):
```

Comment:
I refactored it because the C implementation would RuntimeError. It's renamed to `test_remove_with_clear_children` because `mutating` was also a valid description for the second test. (I just upgraded it actually).

Reply:
NVM, I incorrectly converted the test. I'll revert it.